draft-ietf-quic-http.md

```diff
@@ -1396,22 +1396,25 @@ servers, the initial value of each client setting is the default value.
 For clients using a 1-RTT QUIC connection, the initial value of each server
 setting is the default value. When a 0-RTT QUIC connection is being used, the
 initial value of each server setting is the value used in the previous session.
-Clients MUST store the settings the server provided in the session being resumed
-and MUST comply with stored settings until the current server settings are
-received.  A client can use these initial values to send requests before the
+Clients SHOULD store the settings the server provided in the session being
+resumed and MUST comply with stored settings until the current server settings
+are received.  If the client did not store, or did not receive, server settings
+from the previous session, the initial value of each setting is the default
+value.  A client can use these initial values to send requests before the
```

Comment:
This business about defaults can be split into a new paragraph, ideally one preceding this more complicated part.

> Settings assume default values prior to receiving a SETTINGS frame, or if the SETTINGS frame does not include a value for the setting.  Endpoints can use these default values to send requests before receiving the a SETTINGS frame.